vmm/sandbox/src/vm.rs

```diff
@@ -125,6 +127,7 @@ impl Default for HypervisorCommonConfig {
             image_path: "".to_string(),
             initrd_path: "".to_string(),
             kernel_params: "".to_string(),
+            firmware: "".to_string(),
             enable_mem_prealloc: false,
```

Comment:
Suggestion, not necessary:
```rust
Self {
    debug: false,
    vcpus: 1,
    memory_in_mb: 1024,
    kernel_path: "/var/lib/kuasar/vmlinux.bin".to_string(),
    ..Default::default()
}
```

Reply:
I tried but it caused default recursive call error